src/core/server/http_resources/http_resources_service.test.ts

```diff
@@ -131,7 +130,6 @@ describe('HttpResources service', () => {
             register(routeConfig, async (ctx, req, res) => {
               return res.renderAnonymousCoreApp({
                 headers: {
-                  'content-security-policy': "script-src 'unsafe-eval'",
                   'x-kibana': '42',
```

Comment:
Same as other comment above

```suggestion
                  'content-security-policy': "script-src 'unsafe-eval'",
                  'x-kibana': '42',
```